### PR TITLE
refactor: cleanup cloud_tasks samples

### PR DESCRIPTION
**Pull request by MonochromaticCoder**
## Description

b/280880999

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by MonochromaticCoder** [comment]
The Python 3.7 and 3.8 checks are failing because I added Python 3.9 type hint using built in types (`list` and `dict`). Is there a way to disable these checks or should I use `List` and `Dict` instead?

**Comment by rsamborski** [comment]
> The Python 3.7 and 3.8 checks are failing because I added Python 3.9 type hint using built in types (`list` and `dict`). Is there a way to disable these checks or should I use `List` and `Dict` instead?

I suggest to use old types with `from typing import List` statements. According to https://endoflife.date/python we should be supporting 3.8 at least till 14 Oct 2024.

File: cloud_tasks/snippets/list_queues.py [hunk 1]
```diff
@@ -13,28 +13,31 @@
...
 # [START cloud_tasks_list_queues]
-def list_queues(project, location):
-    """List all task queues."""
+from google.cloud import tasks_v2
 
-    from google.cloud import tasks_v2
 
-    # Create a client.
-    client = tasks_v2.CloudTasksClient()
+def list_queues(project: str, location: str) -> list[str]:
```
Comment (rsamborski) [inline cloud_tasks/snippets/list_queues.py:19]: As mentioned in the other comment please use List to support Python 3.7 and 3.8.

**Review by rsamborski — APPROVED** [review]
Nice work! Thanks.

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**